esmvaltool/cmorizers/obs/cmor_config/PHC.yml

```diff
@@ -0,0 +1,31 @@
+---
+# Common global attributes for Cmorizer output
+attributes:
+  dataset_id: PHC
+  version: '3'
+  tier: 2
+  modeling_realm: clim
+  project_id: OBS6
+  source: 'https://data.nodc.noaa.gov/woa/WOA13/DATAv2/'
```

Comment:
Is this correct?

Reply:
Not at all, thanks for the catch, will change.